src/python/WMCore/WMBS/File.py

```diff
@@ -529,19 +529,18 @@ def addFilesToWMBSInBulk(filesetId, workflowName, files, isDBS=True,
     runLumiBinds = []
     fileCksumBinds = []
     fileLocations = []
-    fileCreate = []
-    fileLFNs = []
-    lfnsToCreate = []
-    lfnList = []
+    fileCreate = set()
+    fileLFNs = set()
```

Comment:
With the merge ACDC files code in place, we shouldn't have any duplicate files, but if we do, these set will save some memory and some queries, since it will have less binds (all those queries use INSERT IGNORE). Unless you think it can harm us in the future, I think we can just keep it as it is(?)

I'm changing `fileCreate` back to list though